### ISSUE TITLE
Cannot load "madelon" dataset

### ISSUE THREAD

**Issue by janfrancu** [aicenter/GenerativeAD.jl#4]
There are no easy anomalies in `madelon` dataset and therefore the loading fails on hcat.

```julia
julia> using GenerativeAD                                                                                             
                                                                                                                      
julia> data = GenerativeAD.load_data("madelon", seed=1)                                                               
ERROR: ArgumentError: number of rows of each array must match (got (0, 500))                                          
Stacktrace:                                                                                                           
 [1] _typed_hcat(::Type{Float32}, ::Tuple{Array{Float32,2},Array{Float32,2}}) at ./abstractarray.jl:1323              
 [2] typed_hcat at ./abstractarray.jl:1310 [inlined]                                                                  
 [3] hcat at /home/honza/Apps/julia-1.4.1/share/julia/stdlib/v1.4/SparseArrays/src/sparsevector.jl:1079 [inlined]     
 [4] load_uci_data at /home/honza/Projects/modules/GenerativeAD.jl/src/data.jl:41 [inlined]                           
 [5] load_data(::String, ::Tuple{Float64,Float64,Float64}; seed::Int64, contamination::Float64, kwargs::Base.Iterators
.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}) at /home/honza/Projects/modules/GenerativeAD.jl/src/data.jl:19
 [6] top-level scope at REPL[2]:1  
```

**Comment by vitskvara** [comment]
Same is true for `gisette`. I suggest excluding these two datasets from the evaluation as they are pretty nasty /large) anyway.

**Comment by janfrancu** [comment]
Resolved with #9
